src/components/PageComponentDetails.js

```diff
@@ -118,10 +118,13 @@ class PageInspect extends Component {
     const { filterOptions, filterValue, definition, curation, harvest } = this.props
     return (
       <Grid className='main-container'>
-        <Row className="show-grid spacer">
-          <Col md={10} mdOffset={1}>
+        <Row className="show-grid well">
```

Comment:
not keen on this being a well. The design is meant to be super clean without boxes, borders, grid lines, ...  Is this needed?

Reply:
No I can remove, it's just that `spacer` had issues. I'll just add more margin between navbar and content to achieve same effect as `spacer`.